test/test-ffmpegCommandService.js

```diff
@@ -45,10 +49,23 @@ describe('FfmpegCommandService', function() {
                             height: 1080
                         }
                     },
-                    workingDirectory: Path.resolve(__dirname, './fixtures/assets1')
-                });
-
-                Should(command).eql('/usr/local/bin/ffmpeg -i ' + Path.resolve(__dirname, './fixtures/assets1/sample.mp4') + ' -filter_complex "[0:v] trim=start=0:duration=600, setpts=PTS-STARTPTS [v0]; [v0] concat=n=1:v=1:a=0 [v_concat]; [v_concat] scale=1920:1080 [v_scaled]" -map "[v_scaled]" -y ' + Path.resolve(__dirname, './fixtures/assets1/output.mp4'));
+                    workingDirectory: _path('./fixtures/assets1')
+                });
+
+                const expected = '/usr/local/bin/ffmpeg ' +
+                    '-i ' + _path('./fixtures/assets1/sample.mp4') +
```

Comment:
I'd love to store these somewhere else...or create custom matchers for them. This is real dirty.